roles/filetree_create/tasks/settings.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Get current Settings from the API
   ansible.builtin.set_fact:
-    changed_settings: "{{ query(controller_api_plugin, 'api/v2/settings/changed/', host=controller_hostname, oauth_token=controller_oauthtoken, verify_ssl=controller_validate_certs,
+    changed_settings: "{{ query(controller_api_plugin, 'api/v2/settings/all/', host=controller_hostname, oauth_token=controller_oauthtoken, verify_ssl=controller_validate_certs,
```

Comment:
should we update the name of the fact? Since it will export all settings, it shouldn't be "changed_settings", should it? 

I'm wondering if we should do the same for the aap_configuration_extended collection.

Reply:
Absolutely true. Changed! And I'm also porting this change to the `infra.aap_configuration_extended` collection as well.